checkbox-ng/checkbox_ng/launcher/subcommands.py

```diff
@@ -295,7 +295,7 @@ def _configure_restart(self, ctx):
             _ = detect_restart_strategy(session_type="local")
         except LookupError as exc:
             _logger.warning(exc)
-            _logger.warning(_("Automatic restart disabled!"))
+            _logger.warning(gettext.gettext("Automatic restart disabled!"))
```

Comment:
I'd not even try to call gettext at all. Translations files for checkbox are completely outdated and i18n support will be dropped soon (next cycle).

Reply:
I think we can let this be for now and once we remove them, we remove them all

Reply:
ok for me.